src/CombFilter/CombFilter.cpp

```diff
@@ -0,0 +1,50 @@
+#include "CombFilter.h"
+
+CCombFilterBase::CCombFilterBase(int iDelayInSamples, int iNumChannels)
+	:m_fGain(1)
+{
+	m_iDelayLength = iDelayInSamples;
+	m_iNumChannels = iNumChannels;
+	pCRingBuff = new CRingBuffer<float>*[m_iNumChannels];
+	for (int i = 0; i < m_iNumChannels; ++i)
+	{
+		pCRingBuff[i] = new CRingBuffer<float>(m_iDelayLength);
```

Comment:
Should reset when user setting delay time, since without resampling/interpolation and keep all of the previously buffered samples is pointless. Will be fixed